### PR TITLE
Updated dblp

### PR DESCRIPTION
**Pull request by fbernhart**
dblp

**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

### CODE REVIEW

File: icons/dblp.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>dblp icon</title><path d="M21.3 14.58l-2.45-9.16s-.15-.59-.72-.33l-5.6 2.6.86 3.2 1.09 4.08.01.06c.05.47-.41.79-.41.79l-1.88.87.5 1.89.04.1c.07.17.28.6.81.91l6.95 4.27s.68.41.52-.17l-1.98-7.4 1.86-.86s.56-.26.4-.85zM13.76 14.67l-.01-.06-1.13-4.22-.66-2.45-.67-2.49v-.04s-.16-.59-.84-1L3.5.14s-.68-.41-.52.17l1.99 7.42-1.87.87s-.56.26-.4.85l2.45 9.16s.16.59.72.33l6.17-2.87 1.3-.61s.52-.24.42-.79z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>dblp icon</title><path d="M21.2995 14.5799l-2.4499-9.1595s-.15-.59-.72-.33L12.53 7.6902l.86 3.1999 1.0899 4.0797.01.06c.05.47-.41.79-.41.79l-1.88.87.5 1.8898.04.1c.07.17.28.6.81.91l6.9496 4.2698s.68.41.52-.17l-1.9799-7.3996 1.86-.86s.5599-.26.3999-.85zm-7.5396.09l-.01-.06-1.13-4.2198-.6599-2.4499-.67-2.4898v-.04s-.16-.59-.84-1L3.5006.1406s-.68-.41-.52.17l1.9899 7.4196-1.87.87s-.5599.26-.3999.85l2.4499 9.1594s.16.59.72.33l6.1696-2.8698 1.3-.61s.5199-.24.4199-.79z"/></svg>
```
Comment (adamrusted) [inline icons/dblp.svg:1]: Looks great! Any chance you can shrink it down to 3 decimals, and also fix the merge conflicts? Saves me butchering it like I did #4025

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/dblp.svg (hunk 1 above)
Comment (fbernhart) [reply icons/dblp.svg:1]: Sure. I was already wondering what was going on in #4025 as I saw something like "269 files changed". 😄

Comment (adamrusted) [reply icons/dblp.svg:1]: I ***really*** did a number on it while trying to merge changes! 😅
Thank goodness for `git reset`!

Comment (fbernhart) [reply icons/dblp.svg]: Fixed! 👍🏼

**Review by adamrusted — APPROVED** [review]
LGTM! 💪🏻

**Pull request merged**